### PR TITLE
SSCS-8741 Include audio video evidence

### PR DESCRIPTION
**Pull request by jackmaloney**
https://tools.hmcts.net/jira/browse/SSCS-8741

- Include audio video evidence in process audio video evidence event
- Change bundle to work with SSCS audio video documents and DWP audio video documents
- Minor refactoring

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java [hunk 1]
```diff
@@ -67,6 +67,10 @@ private void setProcessAudioVideoActionDropdown(SscsCaseData sscsCaseData, boole
 
         populateListWithItems(listOptions, ISSUE_DIRECTIONS_NOTICE);
 
+        if (hasJudgeRole || hasSuperUserRole) {
+            populateListWithItems(listOptions, INCLUDE_EVIDENCE);
```
Comment (ryandac) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java:71]: could put this on line 75

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToSubmitHandler.java [hunk 2]
```diff
@@ -93,6 +94,94 @@ private void processIfIssueDirectionNotice(SscsCaseData caseData) {
...
+    }
+
+    private SscsDocument buildAudioVideoSscsDocument(AudioVideoEvidence audioVideoEvidence, PreSubmitCallbackResponse<SscsCaseData> response) {
+        return SscsDocument.builder().value(
+                SscsDocumentDetails.builder()
+                        .documentLink(audioVideoEvidence.getValue().getDocumentLink())
+                        .documentFileName(audioVideoEvidence.getValue().getFileName())
+                        .documentType(findAudioVideoDocumentType(audioVideoEvidence, response))
+                        .documentDateAdded(audioVideoEvidence.getValue().getDateAdded().toString())
+                        .partyUploaded(audioVideoEvidence.getValue().getPartyUploaded())
```
Comment (ryandac) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToSubmitHandler.java:168]: Not sure i party uploaded is needed, but I think the documentType is needed (i.e. type of evidence)

Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToSubmitHandler.java:168]: This is what we discussed on slack. The document type will be audio and video, and then they'll use the party uploaded field to see who uploaded it.

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java (hunk 1 above)
Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java:71]: Done

**Review by ryandac — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToSubmitHandler.java (hunk 2 above)
Comment (rasikasri) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToSubmitHandler.java:168]: Why are we using enum as it is for some fields (partyUploaded) and string value of enum (documentType) for some other fields?

Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToSubmitHandler.java:168]: Document type should be an enum but it's used all over the place so a big job. We should do this as part of tech debt

**Pull request merged**